app/Jobs/PlatformStatsSummaryJob.php

```diff
@@ -28,17 +30,25 @@
 */
 class PlatformStatsSummaryJob extends Job
```

Comment:
This job now probably requires an increased `timeout` value so that in production it doesn't get killed by the supervisor while still fetching data. Other jobs that poll all wikis currently use a value of `1800` so maybe it's a good idea to use the same one here. E.g.: https://github.com/wbstack/api/blob/5037c4f8ad4923966eba350c7bfa56f812616bef/app/Jobs/PollForMediaWikiJobsJob.php#L13